go.mod

```diff
@@ -120,7 +119,7 @@ require (
 	google.golang.org/grpc v1.45.0 // indirect
 	google.golang.org/protobuf v1.27.1 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
-	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Maybe moving this entry to the "direct" section above this block, where the fork was for better visibility.

Reply:
good call, yes, that's better

Reply:
Ah yeah, good catch - moved it now. For some reason our `go.mod` has 2 indirect sections, so I think `go mod tidy` didn't move things as it normally would 🤔